src/commands/jobs/util.ts

```diff
@@ -53,6 +53,10 @@ export const listOutputFormat = {
 		{
 			header: 'Status',
 			property: 'status'
+		},
+		{
+			header: 'Discriminator',
+			property: 'discriminator'
```

Comment:
Since `discriminator` is so closely related to `name`, maybe it makes sense to order it that way in the table.

Reply:
Good point, I adjusted it.